package.json

```diff
@@ -5,7 +5,9 @@
   "author": "CZI <edu-frontend-infra@chanzuckerberg.com>",
   "homepage": "https://github.com/chanzuckerberg/edu-design-system",
   "license": "MIT",
-  "main": "lib/index.js",
+  "main": "lib/cjs/index.js",
+  "module": "lib/es/index.js",
```

Comment:
These will certainly work 👍 

Although in the future we may want to switch over to [conditional exports](https://nodejs.org/api/packages.html#conditional-exports).

Per https://nodejs.org/api/packages.html#dual-commonjses-module-packages (and other places in the docs), I think `exports` is preferred now (although main/module is still common).